tree/dataframe/test/dataframe_concurrency.cxx

```diff
@@ -10,10 +10,12 @@
 
 #include "gtest/gtest.h"
 
+static const auto NUM_THREADS = 8u;
+
 #ifdef R__USE_IMT
 TEST(RDFConcurrency, NestedParallelismBetweenDefineCalls)
 {
-   ROOT::EnableImplicitMT();
+   ROOT::EnableImplicitMT(NUM_THREADS);
```

Comment:
It is not useless to run with the max number of core when we can.  Could we make this to be set just for the 32 bits build?

Reply:
Yep, I can do that!

Reply:
@pcanal What about this?

Reply:
Wouldn't the test also fails on Linux/32bits ... if there was enough cores?

Reply:
Fails also on 64bit with enough cores ;) What is the magic preproc var for detecting 32bit?

Reply:
Found it, it's `R__B64`

Reply:
Can you `#include "ROOT/RConfig.hxx"` to make sure we see `R__B64` without relying on side-effects of other `#include`s?